src/System.Runtime.WindowsRuntime/src/System/Windows/Size.cs

```diff
@@ -54,7 +53,7 @@ public double Width
             set
             {
                 if (value < 0)
-                    throw new ArgumentOutOfRangeException(SR.ArgumentOutOfRange_NeedNonNegNum, nameof(Width));
```

Comment:
Because the intent was obviously to use the name of the properties, we don't have an established guideline here for corefx, and we're not consistent across corefx. Unless we establish a guideline and enforce it across the repo, I'd rather not change things like this unnecessarily. This code is also only for WinRT and isn't tested by the CI system.